src/workflows-app/RunDetails.js

```diff
@@ -249,10 +278,23 @@ export const BaseRunDetails = (
       () =>
         div([
           div({ style: { padding: '1rem 2rem 2rem' } }, [header]),
-          div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 2rem' } }, [
+          div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 1rem' } }, [
             h(WorkflowInfoBox, { workflow }, []),
             h(TroubleshootingBox, { name, namespace, logUri: workflow.workflowLog, submissionId, workflowId, showLogModal }, []),
           ]),
+          div({ style: { fontSize: 16, padding: '0rem 2.5rem 1rem' } }, [
+            span({ style: { fontWeight: 'bold' } }, ['Approximate workflow cost: ']),
+            renderInProgressElement(workflow),
+            renderTaskCostElement(taskCostTotal),
+            h(
+              TooltipTrigger,
+              {
+                content:
+                  'Approximate cost is calculated based on the list price of the VMs used and does not include disk cost, subworkflow cost, or any cloud account discounts',
+              },
+              [icon('info-circle', { style: { marginLeft: '0.4rem', color: colors.accent(1) } })]
```

Comment:
Given the lack of subworkflows, I might advocate for making this more "warning" than "info-circle". What do you think?

Reply:
I'm torn. My hesitation is around not wanting users to feel like they've done something wrong, or that something is wrong with their tasks/subworkflows. I think for that reason I'd go for the info-circle, but I can be convinced.